HelpSource/Classes/Pfunc.schelp

```diff
@@ -49,3 +60,28 @@ a = Pfunc({ exprand(0.1, 0.3) + #[1, 2, 3, 6, 7].choose }).asStream;
 }.fork;
 )
 ::
+
+When an Event is played, if code::\freq:: is set then code::\degree:: is ignored (due to the code::Event.default:: machinery). In a chain of Patterns, a code::Pfunc:: can be used to delete a key from the Event stream; this can even be done inside a Pbind.
+
+code::
+q = Pbind(\freq, 300, \dur, Pn(0.3, 2));
+q.trace.play;
+
+// Sound-wise ineffective modification of the incoming stream
+p = Pbind(\degree, 6) <> q;
+p.trace.play;
+
+// Instead
+p = Pbind(\degree, 6) <> Pfunc { |ev| ev.freq = nil } <> q;
+p.trace.play;
+
+// Alternatively
+p = Pbind(\degree, Pfunc { |ev| ev.freq = nil; 6 } ) <> q;
+p.trace.play;
+
+// Just setting a key to nil from a Pbind pair will end the stream
+p = Pbind(\freq, nil, \degree, 6) <> q;
+p.trace.play; // silent because it outputs no events
+::
+
+
```

Comment:
please remove the extra newlines at the end of this file

Reply:
Done. Alas forgot to `[skip ci]` this time... (But then, if I had accidentally deleted a `:` it would not have compiled anymore, as I found out in the past. :stuck_out_tongue: That's why I leave the compile on for the help too.)

Reply:
CI doesn't currently build documentation, but we'd like to have that!